### PR TITLE
update to build on Yosemite and run on iOS 8

### PR DESCRIPTION
**Pull request by 100apps**
Keychain-Dumper is an awesome tool for jailbreaked iOS devices. I made some changes to  build on Yosemite and run on iOS 8.
1. changed Makefile to build more easily
2.  fixed [#7](https://github.com/ptoomey3/Keychain-Dumper/issues/7) 
3. add a build and run gif demo in Readme, for new users.

Thank you @ptoomey3

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -30,13 +15,21 @@ LDFLAGS	=\
...
+	@$(GCC_ARM) $(LDFLAGS) main.o -o keychain_dumper
 
 main.o: main.m
 	$(GCC_ARM) -c main.m
 
 clean:
 	rm -f keychain_dumper *.o 
+
+list:
+	security find-identity -pcodesigning -v
```
Comment (ptoomey3) [inline Makefile]: I'm not super familiar with `find-identity`, but it looks like `-v` will only return "valid" code signing certificates. That seems to exclude self-signed certificates (as are used in the rest of the Readme). I'm assuming that self-signed still works fine (give this is being run on a jail broken device), right? If so, can you update this to remove the` -v`?

File: README.md [hunk 2]
```diff
@@ -30,6 +30,9 @@ By default passing no option flags is equivalent to running keychain_dumper with
     
 ## Building
 
+### Demo
+![build and run gif demo](https://raw.githubusercontent.com/100apps/Keychain-Dumper/gh-pages/build-and-run.gif)
```
Comment (ptoomey3) [inline README.md]: Can you pull this gif down locally, add it to the repo, and then update the Readme to have a relative img url that points to it? Something like: https://help.github.com/articles/relative-links-in-readmes/

File: main.m [hunk 3]
```diff
@@ -315,7 +315,8 @@ int main(int argc, char **argv)
 	}
 	
 	NSArray *keychainItems = nil;
-	for (id *kSecClassType in (NSArray *) arguments) {
+//fix https://github.com/ptoomey3/Keychain-Dumper/issues/7
```
Comment (ptoomey3) [inline main.m]: Ah...thanks. Can you remove the comment?

**Comment by ptoomey3** [comment]
Thanks so much for the build update. Once this is ready to merge, I'd like to build it locally, sign it with a self signed cert, and update the binary that is included in the repo. But, I currently have no jailbroken device to test it with. Would you be willing to test it? If not, I totally understand :smile:.

**Comment by ptoomey3** [comment]
Also, I can update the Readme to use the new make build targets after we merge this.

File: main.m (hunk 3 above)
Comment (100apps) [inline main.m]: I've remove the comment

File: README.md (hunk 2 above)
Comment (100apps) [inline README.md]: got it.
What's more, I add [create-certificate.gif](https://github.com/100apps/Keychain-Dumper/blob/master/create-certificate.gif) for freshman.

File: Makefile (hunk 1 above)
Comment (100apps) [inline Makefile]: I've tested on my devices, self-signed certificates do work.

**Comment by 100apps** [comment]
@ptoomey3, I have more than 5 jailbroken devices, ready for your tests. 😄

**Comment by nderkach** [comment]
Works for me with iOS 8.4 built on 10.10.4.

**Comment by ptoomey3** [comment]
Cool. I'm going to clean up a few things and get this merged in the next few days.

**Comment by ptoomey3** [comment]
Closed in preference to https://github.com/ptoomey3/Keychain-Dumper/pull/17

**Pull request closed without merging**